### PR TITLE
[ Rel-5_0 Bug 11575 ] - Logo overwrites toolbar 

### PR DESCRIPTION
**Pull request by s7design**
http://bugs.otrs.org/show_bug.cgi?id=11575

Hi @mrcbnsls 

Here is our proposal for fixing this bug. After testing it we saw that the issue was happened during re-adding #ToolBar. In the Header.tt at first there is #Logo and then #ToolBar. Because of that Logo in slim skins caused such problem. In OTRS4 there is not this problem. 

I saw another problem here. The fix in Core.Agent.Responsive.js 

<pre> $('#Logo').detach().prependTo('#Header'); </pre>
was not enough. On reloading the page, width of navigation bar (z-index=20) is 98% so in slim skins it is above the toolbar menu (z-index=10). In that case it is not possible to click on items of toolbar menu ( actually it is possible on top of any item, if you have enough patience :) )
I saw also that after resizing the screens the menu items are available again. It is solved with Core.Agent.ResizeNavigationBar. However, the problem was with checking the class 'Visible-ScreenXL'. I removed this checking and it works fine now. I don't know is it will cause some bad side effect.

Please, follow up the PR and let me know what is your opinion about that. 

Regards
Zoran

### CODE REVIEW

**Comment by mrcbnsls** [comment]
Hi Zoran (@s7design), thanks for the PR, but I preferred to solve it in another way. Please have a look at the commit IDs from the bug entry. I'll close this PR.
